Source/Flow/Public/Types/FlowDataPinBlueprintLibrary.h

```diff
@@ -0,0 +1,185 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "FlowDataPinProperties.h"
+#include "FlowDataPinResults.h"
+
+#include "FlowDataPinBlueprintLibrary.generated.h"
+
+// Auto-cast operators for blueprint to their inner types
+UCLASS()
+class UFlowDataPinBlueprintLibrary : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
+public:
+
+	// Recommend implementing AutoConvert_FlowDataPinProperty... for every EFlowPinType
+	FLOW_ASSERT_ENUM_MAX(EFlowPinType, 12);
```

Comment:
These static assets will fire at compile-time whenever we add new PinTypes to the enum.  I put them in locations in the code where we will want to at least consider making changes when adding new PinTypes.  This will help call attention to all of the various places in the code that need to be updated when adding values.